client/components/LearnerViewDirective.js

```diff
@@ -318,6 +324,21 @@ tie.directive('learnerView', [function() {
         .tie-previous-instructions {
           opacity: 0.5;
         }
+        .tie-reinforcement li {
+          list-style: none;
+          margin: 0;
+          margin-top: 1px;
+          position: relative;
+        }
+        .bullet-img {
```

Comment:
Could you use `.tie-bullet-img` or `.tie-reinforcement .bullet-img` instead? We prefix all our CSS classes with `.tie-` in order to distinguish them from third-party libs.